client/src/utils/sessionStorageUtils/sessionStorageUtils.js

```diff
@@ -1,6 +1,15 @@
-import initSessionStorage from './sessionStorage';
 import { getAvailableApplications } from '../../api/applicationAndTaskSelection';
 
+/**
+ * @description Checks if passed item already exists in session storage and initializes with given value if not existing.
+ * @param {*} itemToCheckFor The selected item to check for in the session storage.
```

Comment:
As far as I can tell both parameters should never be anything else than a String, so could you please add that as a type